parser/src/meta.ts

```diff
@@ -0,0 +1,9 @@
+import { StringLiteral, TemplateLiteral } from "./scalars";
+import { unionType } from "./unionType";
+import { objectTypeWithKnownKeys } from "./utils";
+
+export const MetaDescription = unionType([StringLiteral, TemplateLiteral]);
+
+export const StateMeta = objectTypeWithKnownKeys({
+  description: MetaDescription,
```

Comment:
Is this on `state.meta`? `description` is now a direct property of the state node, and we shouldn't assume it's on `meta.description` anymore... or maybe we still should. Thoughts welcome

Reply:
We need to be able to parse whatever the user throws at us - the parser should handle not just current XState syntax, but also slightly older stuff too.